src/runtime_src/core/edge/ps_kernels/profiling/aie_profile_config/CMakeLists.txt

```diff
@@ -0,0 +1,41 @@
+# SPDX-License-Identifier: Apache-2.0
+# Copyright (C) 2018-2022 Xilinx, Inc. All rights reserved.
+# Copyright (C) 2022 Advanced Micro Devices, Inc. All rights reserved.
+set(SRC_DIR  "${CMAKE_CURRENT_SOURCE_DIR}")
+set(PS_KERNEL_INSTALL_DIR "${XRT_INSTALL_LIB_DIR}/ps_kernels_lib")
+
+include_directories(
+  ${SRC_DIR} 
+  ${SRC_DIR}/../../../include
+  ${SRC_DIR}/../../../../
+  ${SRC_DIR}/../../../edge/include
+  )
+
+file(GLOB SRC_FILES
+  "${SRC_DIR}/*.cpp"
+  )
+
+set(CMAKE_CXX_FLAGS "-DXAIE_DEBUG -D__PS_ENABLE_AIE__ ${CMAKE_CXX_FLAGS}")
```

Comment:
Don't change CMAKE flags, add the defines to the target using `target_compile_defintions`